### PR TITLE
Fix: update password, profile update, image from API and remove not set string for values that are missing

### PR DESCRIPTION
**Pull request by mhizterkeyz**
# Description

Fix: update password, profile update, image from API and remove not set string for values that are missing

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnin

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
         "lodash": "^4.17.13",
         "resolve-url-loader": "^2.3.1",
         "sass": "^1.15.2",
-        "sass-loader": "^7.1.0"
+        "sass-loader": "^7.1.0",
+        "vue-template-compiler": "^2.6.11"
```
Comment (Jeremiahiro) [inline package.json:20]: why did you add these?

We are not using Vue in this project and you jus added Vue compiler

Comment (mhizterkeyz) [reply package.json:20]: The Vue compiler came with the last pull I did. I didn't even install any node dependencies
